docs/standard/frameworks.md

```diff
@@ -1,56 +1,126 @@
-
-## Framework Versions
-
-The table below defines the set of frameworks that you can use, how they are referred to and which version of the [.NET Standard Library](library.md) that they implement.
-
-| Framework | Latest Version | Target Framework Moniker (TFM) | Compact Target Framework Moniker (TFM) | .NET Standard Version | Metapackage |
-|:--------: | :--: | :--: | :--: | :--: | :--: | :--: |
-| .NET Standard | 1.6 | .NETStandard,Version=1.6 | netstandard1.6 | N/A | [NETStandard.Library](https://www.nuget.org/packages/NETStandard.Library)|
-| .NET Core Application | 1.0.1 | .NETCoreApp,Version=1.0 | netcoreapp1.0 | 1.6 | [Microsoft.NETCore.App](https://www.nuget.org/packages/Microsoft.NETCore.App)|
-| .NET Framework | 4.6.2 | .NETFramework,Version=4.6.2 | net462 | 1.5 | N/A |
-
-> [!NOTE]
-> These framework versions are the latest stable versions. There may be pre-released versions as well that are not described by this table.
-
-## Writing about Frameworks
-
-There are multiple ways to refer to frameworks in written form, most of which are used in this documentation. They are described below, both as a legend for interpreting the documentation but also to guide use in other documents.
-
-Using .NET Framework 4.6.1 as an example, the following forms can be used:
-
-**Referring to a product**
-
-You can refer to a .NET platform or runtime.
-
-- ".NET Framework 4.6.1"
-
-**Referring to a Framework**
-
-You can refer to a framework or targeting of a framework using long- or short-forms of the TFM. Both are equally valid in the general case.
-
-- `.NETFramework,Version=4.6.1`
-- `net461`
-
-**Referring to a family of Frameworks**
-
-You can refer to a family of frameworks using long- or short-forms of the framework ID. Both are equally valid in the general case.
-
-- `.NETFramework`
-- `net`
+# Target frameworks
+
+The .NET ecosystem has a concept of *target frameworks*. Frameworks define the API that you can use to target a particular platform. For example, the .NET Framework 4.6 is a platform that you can target. Visual Studio and other IDEs and editors use frameworks to provide you with the correct set of APIs. NuGet also uses them to ensure that you produce and consume appropriate NuGet packages and underlying assets for the framework you're targeting.
+
+| Reference | Name                                      |
```

Comment:
I thought this was you rewriting the Writing about Frameworks section which is shown on the diff, no?